requirements.txt

```diff
@@ -1,6 +1,6 @@
 crython
-suds
 requests
 sentry_sdk[flask]
 PyGithub
-jinja2
\ No newline at end of file
+jinja2
+git+git://github.com/betheluniversity/cascade.git
```

Comment:
you can just use bu-cascade (or possibly bu_cascade?) instead of `git+git://github.com/betheluniversity/cascade.git`